app/services/kredits.js

```diff
@@ -116,6 +116,14 @@ export default Service.extend({
     })
   }),
 
+  totalKreditsEarned: computed(function() {
+    return this.kredits.Contribution.functions.totalKreditsEarned(true)
```

Comment:
Not a fan of the unnamed `true` in this new function call, because the meaning is completely hidden, until one finds and reads the Solidity source code of it.

Reply:
yes, me neither. but it is a contract call and generated from the wrapper. 
could at some write our own function in the wrapper class.

Reply:
Ah, yes.